docs/source/tutorial/introduction.md

```diff
@@ -1,74 +1,80 @@
 ---
 title: 0. Introduction
-description: Start here to learn how to build fullstack apps with Apollo
+description: Start here to learn how to build full-stack apps with Apollo
```

Comment:
I'm still not used to seeing "full-stack/fullstack/full stack" hyphenated. I know this is in our style guide now -- any reason for hyphenating?

Reply:
"Full stack" is a two-word adjectival phrase that precedes the noun that it modifies ("apps"). According to the Chicago Manual of Style, such phrases should be hyphenated